internal/pkg/node/datastructure.go

```diff
@@ -75,27 +79,24 @@ type KernelConf struct {
 
 type NetDevs struct {
 	Type    string            `yaml:"type,omitempty" lopt:"type" sopt:"T" comment:"Set device type of given network"`
-	OnBoot  string            `yaml:"onboot,omitempty" lopt:"onboot" comment:"Enable/disable network device (true/false)" type:"bool"`
+	Manual  bool              `yaml:"manual,omitempty" lopt:"manual" comment:"Enable/disable network device (true/false)"`
 	Device  string            `yaml:"device,omitempty" lopt:"netdev" sopt:"N" comment:"Set the device for given network"`
 	Hwaddr  string            `yaml:"hwaddr,omitempty" lopt:"hwaddr" sopt:"H" comment:"Set the device's HW address for given network" type:"MAC"`
-	Ipaddr  string            `yaml:"ipaddr,omitempty" comment:"IPv4 address in given network" sopt:"I" lopt:"ipaddr" type:"IP"`
-	IpCIDR  string            `yaml:"ipcidr,omitempty"`
```

Comment:
It's removed in the config in order to avoid conflicting configurations of Ipaddr, Netmask and IpCIDR.
BUT for templates you have 
```
/*
Return the ipv4 address and mask in CIDR format. Aimed for the use in
templates.
*/
func (netdev *NetDevs) IpCIDR() string {
	if netdev.Ipaddr.IsUnspecified() || netdev.Netmask.IsUnspecified() {
		return ""
	}
	ipCIDR := net.IPNet{
		IP:   netdev.Ipaddr,
		Mask: net.IPMask(netdev.Netmask),
	}
	return ipCIDR.String()
}
```
so it's not removed.